update examples to linopy framework (#591)
PyPSA/PyPSA: doc/examples-lopf.rst

#########################

> This section contains examples of network optimization with PyPSA. Note that PyPSA allows you to optimize networks in three ways: based on Linopy, based on Pyomo, and based on custom code. PyPSA's default optimization framework is based on Linopy. In earlier versions, PyPSA used the ``pyomo`` interface and an in-house implementation, both available through the `lopf` function. The latter are still available, but will be deprecated in the future. The following examples all use the Linopy-based implementation.

Reviewer: ```suggestion
This section contains examples of network optimization with PyPSA. PyPSA allows you to optimize networks in three ways: with `Linopy <https://linopy.readthedocs.io>`_, with `Pyomo <http://www.pyomo.org/>`_, and based on `custom code <https://github.com/PyPSA/PyPSA/blob/v0.22.1/pypsa/linopt.py>`_. The new default optimization framework is based on Linopy. In earlier versions, PyPSA used the ``pyomo`` interface and an in-house implementation, both available through the ``n.lopf()`` function. The previous options are still available but will be deprecated in the future. The following examples all use the Linopy-based implementation.
```

.. toctree::
    :maxdepth: 1

    examples/optimization-with-linopy.ipynb

>     examples/lopf_with_pyomo_False.ipynb

Reviewer: This is removed, right?
